### PR TITLE
Replicate production env to AKS

### PR DESCRIPTION
**Pull request by johnake**
### Context

Replicate production env to AKS

### Changes proposed in this pull request

Add workflow for production_aks 
Add prod to db backup and restore workflow

### Checklist

- [ ] Attach to Trello card
- [ ] Rebased main
- [ ] Cleaned commit history
- [ ] Tested by running locally

### CODE REVIEW

**Comment by johnake** [comment]
[Find a lost TRN - Replicate Production environment to AKS](https://trello.com/c/wML4BHbX/708-find-a-lost-trn-replicate-production-environment-to-aks)

File: terraform/aks/workspace_variables/production_aks.tfvars.json
Comment (RMcVelia) [inline terraform/aks/workspace_variables/production_aks.tfvars.json:1]: need to set worker replicas to 0 for production, unless we are certain that the jobs will not affect real prod. Also remove the notify api Prod aks key secret so we don't send any notifications.

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -119,6 +119,29 @@ jobs:
           environment: ${{ matrix.environment }}
           azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
 
+  deploy_v2_production:
+    name: Deploy to production_aks environment
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    environment:
+      name: production_aks
+      url: ${{ steps.deploy.outputs.environment_url }}
```
Comment (RMcVelia) [inline .github/workflows/build-and-deploy.yml:128]: steps.deploy_v2 i would think

File: .github/workflows/build-and-deploy.yml [hunk 2]
```diff
@@ -119,6 +119,29 @@ jobs:
...
+    concurrency: deploy_production_aks
+    needs: [docker, deploy_v2_nonprod]
+
+    outputs:
+      environment_url: ${{ steps.deploy.outputs.environment_url }}
```
Comment (RMcVelia) [inline .github/workflows/build-and-deploy.yml:133]: steps.deploy_v2 I would think

File: .github/workflows/build-and-deploy.yml [hunk 3]
```diff
@@ -119,6 +119,29 @@ jobs:
           environment: ${{ matrix.environment }}
           azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
 
+  deploy_v2_production:
+    name: Deploy to production_aks environment
+    runs-on: ubuntu-latest
```
Comment (RMcVelia) [inline .github/workflows/build-and-deploy.yml:124]: add continue-on-error: true (until migrated)

File: .github/workflows/database-copy.yml [hunk 4]
```diff
@@ -10,7 +10,7 @@ jobs:
     name: Restore DB
     strategy:
       matrix:
-        environment: [dev, test, preprod]
+        environment: [dev, test, preprod, prod]
```
Comment (RMcVelia) [inline .github/workflows/database-copy.yml:13]: shouldn't this be production, not prod

File: .github/workflows/restore-paas-db-to-aks.yml [hunk 5]
```diff
@@ -13,6 +13,7 @@ on:
           - dev
           - test
           - preprod
+          - prod
```
Comment (RMcVelia) [inline .github/workflows/restore-paas-db-to-aks.yml:16]: shouldn't this be production, not prod

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**